tools/portlet_spring_mvc_tmpl/docroot/WEB-INF/jsp/init.jspf

```diff
@@ -0,0 +1,20 @@
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
```

Comment:
Can we simplify the template to only include a single jsp file instead of a file and a fragment?  that will make it more consistent with other templates.